numpy/f2py/tests/test_crackfortran.py

```diff
@@ -33,3 +35,50 @@ def test_module(self):
         self.module.subc([w, k])
         assert_array_equal(k, w + 1)
         assert self.module.t0(23) == b'2'
+
+class TestPublicPrivate():
+    def test_defaultPrivate(self):
+        with tempfile.NamedTemporaryFile() as ff:
+            ff.write(b"""
+module foo
+  private
+  integer :: a
+  public :: setA
+contains
+  subroutine setA(v)
+    integer, intent(in) :: v
+    a = v
+  end subroutine setA
+end module foo
+""")
+            ff.seek(0)
```

Comment:
Does this do anything? If you're passing `crackfortran` a filename, it shouldn't care about the file handle state, right?

Reply:
Yes, but in my tests, the file is empty and not flushed when crackfortran is running, and the returned list is empty. Does `flush()` exist ?

Reply:
Ok, `flush()` exists and do the job. I've changed the code for it, because it makes the intention more clear I guess.